infra/scripts/test-end-to-end.sh

```diff
@@ -53,7 +104,12 @@ ORIGINAL_DIR=$(pwd)
 cd tests/e2e
 
 set +e
-pytest redis/* --junitxml=${LOGS_ARTIFACT_PATH}/python-sdk-test-report.xml
+if [[ ${PYTEST_MARK} = "auth" ]]; 
+then
+	pytest redis/* -m ${PYTEST_MARK} --enable_auth=True --junitxml=${LOGS_ARTIFACT_PATH}/python-sdk-test-report.xml
```

Comment:
can't we share the same command with just values set differently.  I don't see the mark actually used in the tests.  Is it needed?

Reply:
markers doesn't work on fixtures, and client creation is the only difference between the test cases. It would have been perfect if it did. I needed a flag, I could just pass true or false. I added marker so that if we write auth specific test cases in future, we can use the marker.